gradle/script/versions.gradle

```diff
@@ -40,5 +40,5 @@ ext {
     }
     kotlin_version = "1.3.41"
     kotlin_coroutines = "1.3.0-RC"
-    arcgis = "100.7.0"
+    arcgis = "100.8.0-+"
```

Comment:
@puneet-pdx - I would go with a specific daily build, e.g. latest good. If we need a later build to do some development we can always increment.

Reply:
The `minSdk` version needs to increase to 21

Reply:
Could you also remove `buildTools`, which is currently used [here](https://github.com/Esri/arcgis-runtime-toolkit-android/blob/master/arcgis-android-toolkit/build.gradle#L31). We should just use the default based on this document:
https://developer.android.com/studio/releases/build-tools

Reply:
Could you check if we can upgrade `androidGradlePlugin` to 3.6.0. It would require Gradle to be upgraded to 5.6.4 or higher. If this causes issues just create a separate issue and we do this as a separate task when appropriate.

More info here:
https://developer.android.com/studio/releases/gradle-plugin

Reply:
@gunt0001 I have created an issue to address the fourth point

https://devtopia.esri.com/runtime/java/issues/2531